localization/pose_initializer/CMakeLists.txt

```diff
@@ -13,6 +13,7 @@ ament_auto_add_executable(pose_initializer_node
   src/pose_initializer/ndt_module.cpp
   src/pose_initializer/stop_check_module.cpp
   src/pose_initializer/localization_trigger_module.cpp
```

Comment:
I think it would be better to rename it to `ekf_localization_trigger_module` to differentiate it from `ndt_localization_trigger_module`.

Reply:
Thanks for the advice. Fixed.